subworkflows/local/bam_extract_region_samtools/meta.yml

```
@@ -0,0 +1,79 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "BAM_EXTRACT_REGION_SAMTOOLS"
+description: |
+  Subworkflow to extract a list of region from a BAM file using samtools.
+  Each region is first extracted from the BAM file and then merged into a
+  single BAM file.
+keywords:
+  - BAM
+  - region
+  - extract
+  - samtools
+components:
+  - samtools/view
+  - samtools/merge
+  - samtools/index
+input:
+  - ch_bam:
+      description: Channel with input data
+      structure:
+        - meta:
+          type: map
+          description: Metadata map
+        - bam:
+          type: File
```

```suggestion
          type: file
```

Everywhere, check other types to o